server/app.ts

```diff
@@ -25,6 +25,7 @@ import HmppsAuthService from './services/hmppsAuthService'
 import passportSetup from './authentication/passport'
 import AssessRisksAndNeedsService from './services/assessRisksAndNeedsService'
 import ControllerUtils from './utils/controllerUtils'
+import serviceUnavailableConfig from './service-unavailable-config.json'
```

Comment:
given it's a generic timed message, should we use a more generic name, e.g. broadcastMessages?

Reply:
Yeah, I was really struggling to come up with a generic name (initially went for `userNotificationBanner/Message` but that felt like it clashed with our usual `notificationBanner` - I think `broadcastMessage` sounds quite nice 👍